scripts/generateExamplesTests.js

```diff
@@ -90,18 +90,20 @@ function fixupExamples(examples) {
           const goldenFilePath = `./testdata/${jobKey}_comment_golden.md`;
           const commentArgs = step.run
             .replace(/\\/g, '')
+            .replace(/--pull-request \$\{\{github\.event\.pull_request\.number\}\}/g, '--pull-request 1')
```

Comment:
why are we changing this to a constant?

Reply:
It also runs on master, the PR number is empty, but the value is required, thus it throws an error. I think we still want to run it on master, and having the #1 should be enough to check the API fetch.